src/pages/TermsOfService.test.ts

```diff
@@ -15,11 +15,33 @@ jest.mock('react-notifications-component', () => {
   };
 });
 
+type NavExports = typeof import('src/libs/nav');
+jest.mock(
+  'src/libs/nav',
+  (): NavExports => ({
+    ...jest.requireActual<NavExports>('src/libs/nav'),
+    goToPath: jest.fn(),
+  })
+);
+type AuthExports = typeof import('src/libs/auth');
+jest.mock(
+  'src/libs/auth',
+  (): AuthExports => ({
+    ...jest.requireActual('src/libs/auth'),
+    signOut: jest.fn(),
+  })
+);
+
 const setupMockAjax = (termsOfService) => {
   const getTos = jest.fn().mockReturnValue(Promise.resolve('some text'));
   const getTermsOfServiceComplianceStatus = jest.fn().mockReturnValue(Promise.resolve(termsOfService));
   const getStatus = jest.fn().mockReturnValue(Promise.resolve({}));
-  Ajax.mockImplementation(() => ({
+  const acceptTos = jest.fn().mockReturnValue(Promise.resolve({ enabled: true }));
```

Comment:
FYI, there's a shorthand for this:

```suggestion
  const acceptTos = jest.fn().mockResolvedValue({ enabled: true });
```

https://jestjs.io/docs/mock-function-api#mockfnmockresolvedvaluevalue